samples/python_interop/teleportation_project/src/TeleportOperations/Teleport.qs

```diff
@@ -1,16 +1,13 @@
-namespace TeleportLib {
+import TeleportOperations.PrepareState.PrepareState.*;     // references the PrepareState.qs file
```

Comment:
This is preexisting so don't worry about it if you don't want to touch it in this PR - but I feel like the `PrepareState` subfolder can be removed from this project, it only has 1 file in it and adds an unnecessary level to the namespace hierarchy.

Reply:
Since it is a sample about project structure, it makes sense to me that the sample demos sub-folders in projects.

Reply:
Oh, I missed that. I don't think it's particularly valuable to demo subfolders since the "point" of the sample is Python interop, but you're right the README does mention it so might as well keep it consistent 🤷‍♀️